### PR TITLE
XGBoost Model - Hyperparameters

### PR DESCRIPTION
**Pull request by FedericoDM**
## Checklist before requesting a review

- [x] The code runs successfully.

```
Here is some console output
```
- [x] I ran `make lint` and have made the bulk of changes that it has requested.

### CODE REVIEW

File: supreme_court_predictions/models/xg_boost.py [hunk 1]
```diff
@@ -70,24 +85,28 @@ def create(self, df):
...
         vectorize_document = df.loc[:, "tokens"].apply(" ".join)
         bag_of_words_x = vectorizer.fit_transform(vectorize_document)
         bag_of_words_y = df.loc[:, "win_side"]
 
         X_train, X_test, y_train, y_test = train_test_split(
             bag_of_words_x,
             bag_of_words_y,
-            test_size=0.20,
+            test_size=self.test_size,
             random_state=123,
```
Comment (michplunkett) [inline supreme_court_predictions/models/xg_boost.py:99]: Can you set this to the constant?

**Review by michplunkett — CHANGES_REQUESTED** [review]
Smöl change.

**Review by JJModern — APPROVED** [review]
looks good

File: supreme_court_predictions/models/xg_boost.py (hunk 1 above)
Comment (FedericoDM) [reply supreme_court_predictions/models/xg_boost.py:99]: ok yes sure

File: supreme_court_predictions/models/xg_boost.py [hunk 2]
```diff
@@ -70,24 +85,28 @@ def create(self, df):
...
+        )
         vectorize_document = df.loc[:, "tokens"].apply(" ".join)
         bag_of_words_x = vectorizer.fit_transform(vectorize_document)
         bag_of_words_y = df.loc[:, "win_side"]
 
         X_train, X_test, y_train, y_test = train_test_split(
             bag_of_words_x,
             bag_of_words_y,
-            test_size=0.20,
+            test_size=self.test_size,
```
Comment (FedericoDM) [inline supreme_court_predictions/models/xg_boost.py:98]: Here I am using the constant declared as attribute in the class, or is there something I am missing?

Comment (michplunkett) [reply supreme_court_predictions/models/xg_boost.py:98]: It's for the random seed.

File: supreme_court_predictions/models/xg_boost.py [hunk 3]
```diff
@@ -96,7 +96,7 @@ def create(self, df):
             bag_of_words_x,
             bag_of_words_y,
             test_size=self.test_size,
-            random_state=123,
+            random_state=SEED_CONSTANT,
```
Comment (FedericoDM) [inline supreme_court_predictions/models/xg_boost.py:99]: Done @michplunkett ;) I had missed that tiny bit

Comment (michplunkett) [reply supreme_court_predictions/models/xg_boost.py:99]: Ahhhhh yeah!

**Review by michplunkett — APPROVED** [review]
